item-descriptions.lua

```diff
@@ -30,7 +30,7 @@ if not moduleMode then
     print("scripts/item-descriptions.lua is a content library; calling it does nothing.")
 end
 
-local help --[[
```

Comment:
Heh, it's supposed to be `local help = [====[`, then `]====]` at the end. But since this script doesn't even attempt to print help text, I would just move any useful content from here to the documentation above and get rid of the rest.

Reply:
I wasn't sure what the deal with this was, because it was one of the things that had changed between the latest master branch and the revision of the file I was editing (because being a dummy I didn't pull before working on the file). I ended up just going with what was currently in.